FreeRTOS/Test/CMock/stream_buffer/stream_buffer_utest.c

```diff
@@ -0,0 +1,1107 @@
+/*
```

Comment:
Please add @ coverage tags to this file so that the lcov / lcovhtml targets will work correctly.

Reply:
This is fixed in coverage filter script. So not required.